spec/factories/conversations.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+# == Schema Information
+#
+# Table name: conversations
+#
+#  id         :bigint           not null, primary key
+#  name       :string
+#  created_at :datetime         not null
+#  updated_at :datetime         not null
+#
+# Indexes
+#
+#  index_conversations_on_name  (name) UNIQUE
+#
+FactoryBot.define do
+  factory :conversation do
+    name { Faker::Lorem.word }
+  end
```

Comment:
Please add more traits :)

Reply:
What traits do you mean?

Reply:
For example:
In your spec/models/conversation_spec.rb file
On line 24 there is a line
conversation.name = nil
take it out to trait and then use